utils.go

```diff
@@ -12,7 +12,7 @@ import (
 // buildURI assembles the base path and queries.
 func buildURI(path string, options interface{}) string {
 	v, _ := query.Values(options)
-	return (&url.URL{Path: path, RawQuery: v.Encode()}).String()
+	return path + (&url.URL{RawQuery: v.Encode()}).String()
```

Comment:
sorry, my comment must not have been clear. i'm still not sure if we want to do this; it was more of a discussion point. i'm looking into the drawbacks before we swap everything over given most methods use this now and i wouldn't want to make `buildURI` less robust for this single (sad) URL.

Reply:
Ok. Let's do it in a safe way.

Reply:
Definitely changing buildURI may lead to unpredictable bugs in other places